UI-MVC/Controllers/ProjectController.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using BL;
+using DAL.Data_Transfer_Objects;
+using Domain.Identity;
+using Domain.Projects;
+using Domain.Users;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using UIMVC.Models;
+
+namespace UIMVC.Controllers
+{
+    public class ProjectController : Controller
+    {
+        private ProjectManager _mgr;
+        private PlatformManager _mgrPlatform;
+        private readonly UserManager<UIMVCUser> _userManager;
+
+        public ProjectController(UserManager<UIMVCUser> userManager)
+        {
+            _mgr = new ProjectManager();
+            _mgrPlatform = new PlatformManager();
+            _userManager = userManager;
+        }
+
+        #region Index
+
+
+        // GET /Project
+        /*[Route("Project/Index/{id}")]
+        public IActionResult Index(int id)
+        {
+            Project project = _mgr.GetProject(id, false);
+            if (project == null)
+            {
+                return RedirectToAction("HandleErrorCode", "Errors", 404);
+            }
+
+            return View(project);
+        }*/
+
+        /*[HttpPost]
+        public ActionResult Index(Project emp, List<Phase> dept)
+        {
+            emp.Phases = dept;
+            return View(emp);
+        }*/
+
+        #endregion
+
+
+        #region Create
+
+        [Authorize]
+        [HttpGet]
+        public IActionResult Create(int platform)
+        {
+            ViewData["platform"] = platform;
+            return View();
+        }
+
+        [Authorize]
+        [HttpPost]
+        public IActionResult Create(ProjectViewModel pvm, int platform)
+        {
+            UIMVCUser projectUser = _userManager.GetUserAsync(HttpContext.User).Result;
+
+            if (pvm == null)
+            {
+                return BadRequest("Project cannot be null");
+            }
+
+            Project pr = new Project()
+            {
+                User = projectUser,
+                CurrentPhase = pvm.Project.CurrentPhase,
```

Comment:
currentphase wordt niet gevuld de id in de db is 0.

Reply:
verder in de code staat er een foreach loop = daar wordt fases aangepakt in de databank en wordt de id meegegeven en gekoppelt

Reply:
ik spreek hier niet over de hele lijst. Ik spreek hier over de currentphase.

![image](https://user-images.githubusercontent.com/44059248/57237070-b177d700-7026-11e9-81b6-3321a3abf414.png)